litgpt/data/__init__.py

```diff
@@ -30,6 +31,7 @@
     "LongForm",
     "OpenWebText",
     "SFTDataset",
+    "TextFiles",
```

Comment:
We will soon have the problem that it is not clear which data modules are for finetuning and which for pretraining. Let the spirits of software engineering lead us on the path towards the frameworkification of LitGPT 😄

Reply:
I agree. The pretrain and full finetuning use cases in your code base are very similar the main difference I believe is that the full finetuning expects users to provide an instruction-like data. 

It would be nice if you could explicitly point out in your tutorials. Keep up the great work folks! 💯

Reply:
Yes, ideally we only want the SFT finetuning datasets to show up in `litgpt finetune --help` and only the pretraining datasets to show up in `litgpt finetune --pretrain`. I believe we were thinking about that a while back but there was no good way to do that with jsonargparse. I can open a new issue so we don't forget to revisit this some day.